layersvt/device_simulation.cpp

```diff
@@ -1692,7 +1695,11 @@ VKAPI_ATTR VkResult VKAPI_CALL EnumerateDeviceExtensionProperties(VkPhysicalDevi
 
     if (result == VK_SUCCESS && !pLayerName && emulatePortability.num > 0 &&
         !PhysicalDeviceData::HasSimulatedOrRealExtension(physicalDevice, VK_KHR_PORTABILITY_SUBSET_EXTENSION_NAME)) {
-        *pCount += 1;
+        if (!pdd->device_extension_properties_query_) {
+            *pCount += 1;
```

Comment:
In general I guess we can assume that no other layers are modifying this returned count?